src/atoms/forms/radio-button-input.tsx

```diff
@@ -0,0 +1,64 @@
+import * as React from "react";
+import { forwardRef } from "react";
+
+const COMPONENT_CLASS = "c-radio";
+
+export interface RadioButtonProps {
+    autofocus?: boolean;
+    checked: boolean;
+    children?: React.ReactNode;
```

Comment:
Nitpick - consider refactoring to wrap this interface with React's `PropsWithChildren`